### PR TITLE
APP-4297: removed noisy logs, added additional logs

### PR DESCRIPTION
**Pull request by ohEmily**
[APP-4297](https://viam.atlassian.net/browse/APP-4297)

[APP-4297]: https://viam.atlassian.net/browse/APP-4297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: rpc/server_auth.go [hunk 1]
```diff
@@ -359,42 +359,20 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 			break
 		}
 	}
-	if !audVerified {
-		// TODO cleanup at a later date -- urgent issue
-		audVerified = true
```
Comment (dgottlieb) [inline rpc/server_auth.go:364]: So we've understood and patched the underlying problem that necessitated this?

**Comment by dgottlieb** [comment]
Random, but I assume we can close out APP-4275

**Comment by ohEmily** [comment]
@dgottlieb oh good call. For now I'll mark it as in progress so I don't forget to bump app

**Comment by dgottlieb** [comment]
Oops, and I forgot: I think this let's us re-enable the tests skipped in [this commit](https://github.com/viamrobotics/goutils/commit/0d3d5d7470ff04b0cc9a06684fe877266bdc1482).

File: rpc/server_auth.go [hunk 2]
```diff
@@ -404,11 +382,17 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 
 	err = claims.Valid()
 	if err != nil {
+		ss.logger.Errorw("invalid claims",
+			"error", err,
+			"registeredClaims", claims.RegisteredClaims)
```
Comment (edaniels) [inline rpc/server_auth.go:387]: nit: registered_claims (snake case) for structured fields

**Review by edaniels — APPROVED** [review]

File: rpc/server_auth.go (hunk 2 above)
Comment (ohEmily) [reply rpc/server_auth.go:387]: thank you! love these kinds of nits :)

**Pull request merged**